mne/gui/_coreg_gui.py

```diff
@@ -159,6 +180,38 @@ class CoregModel(HasPrivateTraits):
     points_eval_str = Property(depends_on='point_distance')
 
     @cached_property
+    def _get_points(self):
+        with warnings.catch_warnings(record=True):  # comp to None in Traits
+            if self.points_filter is None:
+                self.points_filter = \
+                    np.ones(len(self.hsp.inst_points), np.bool8)
```

Comment:
Yeah hmmm I agree it's hard to find a solution that feels clean...  making `omit_hp_points` aware of the EEG selector state might be the best option (even though not perfect). To make it more transparent, maybe `_get_points()` could also assign a `Property` that is an index to project `points` back to the raw points. For example, with 3 points of the kind `[HSP, EEG, HSP]` the back-projector could be `[0, 2]`.